.github/workflows/ci.yml

```diff
@@ -14,11 +14,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@master
```

Comment:
Perhaps we should use `@stable` here instead of `@master`.
At least that's [what Bevy is currently doing](https://github.com/bevyengine/bevy/blob/3ead10a3e0ab1c23d8e4b6cc9fa8be1eef8c581f/.github/workflows/ci.yml#L32)

Reply:
Using `dtolnay/rust-toolchain@stable` is just a shortcut for
```
uses: dtolnay/rust-toolchain@master
with:
    toolchain: stable
```
which is what the current changes in this PR do. `dtolnay/rust-toolchain` has the feature that you can basically specify the Rust version via the version of the action, if it is not specified via the `toolchain` parameter. Or in other words: Using `dtolnay/rust-toolchain@stable` is just another way of saying _"Please install and use the current stable Rust toolchain."_, and that is what both variants do here.